files/jira.rb

```diff
@@ -4,6 +4,12 @@
 
 class Jira < BaseHandler
 
+  def build_labels
+    [ "SENSU_#{@event['client']['name']}",
+      "SENSU_#{@event['check']['name']}",
+      "SENSU", *@event['check']['tags'] ].uniq
```

Comment:
What is the significance of the asterisk here?

Hmm. splat? I've never seen this. TIL.